R/data_manipulation.R

```diff
@@ -0,0 +1,89 @@
+#' Calculates the sentiment values for each sentences on by one
+#' 
+#' @param sentences vector of strings for which the sentiment values
+#' should be calculated
+#' 
+#' @return vector of sentiment values corresponding to the inout
+#' sentences
+#' 
+#' @import dplyr
+#' @import stringr
+#' @import textdata
+#' @import tidytext
+calculate_sentiment <- function(sentences) {
+  data(stop_words, package = "tidytext")
+  sentiment_index <- get_sentiments("afinn")
+  
+  calculate_single_sentence_sentiment <- function(sentence) {
+    words <- sentence %>% 
```

Comment:
This could be moved to new method, because it is also used in 49 line.

Reply:
Good point. Fixed.